test/system/090-events.bats

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# tests for podman events functionality
+#
+
+load helpers
+
+@test "events with a filter by label" {
+    rand=$(random_string 30)
+    run_podman 0  run --label foo=bar --name test-$rand --rm $IMAGE ls
+    run_podman 0 events --filter type=container --filter container=test-$rand --filter label=foo=bar --filter event=start --stream=false
+    is "$output" ".*foo=bar" "check for label event on container with label"
```

Comment:
You don't need the `0`, it's implicit. And I'd prefer a label of `foo=$random_label`, because it's just too easy for `foo=bar` to (e.g.) trigger on a previously-run container or something like that. But you done good :-), thank you for writing a test

Reply:
I believe that having a random container name does the same thing.  This event looks for a start event on container with a label of foo=bar on a unique.

Reply:
Right -- assuming that the container filtering code is working as intended. (I get paid to be paranoid). I'll fix this in a followup PR.